### DISCUSSION TITLE
Please allow display of sunshine video stream in sunshine web interface

### DISCUSSION THREAD

**Discussion by batchfileframework** [LizardByte/community#333] [Sunshine]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

Due to network conditions, 
I can access my remote computer's sunshine web interface however the moonlight native client cannot communicate to it. 

One solution with MANY side benefit for all sorts of other users 

would be to display the video stream right in the web interface. 

This would allow using any regular browser as a client for sunshine video streaming.

### Expected Behavior

A new tab in the web interface, that shows the current screen

### Host Operating System

Windows

### Operating System Version

10

### Architecture

64 bit

### Sunshine commit or version

0.21

### Package

Windows - portable

### GPU Type

none (software encoding)

### GPU Model

No GPU/laptop

### GPU Driver/Mesa Version

None

### Capture Method (Linux Only)

not on linux

### Config

```Shell
unavailable
```

### Relevant log output

```shell
unavailable
```

**Comment by jimmt41** [comment]
I wonder if it would be better to put in a request to the Moonlight streaming project have a moonlight browser plugin?

**Reply by shodanx2** [reply]
I imagine that the server could push frames directly to the browser easier using existing streaming component like webrtc or what is used by novnc and apache guacamole, than taking the gamestream format, parsing it in browser and then displaying it.

**Reply by shodanx2** [reply]
But any kind of gamestream to web bridge would be awesome too. However I think it would end up adding a third component to the setup

**Reply by ReenigneArcher** [reply]
https://github.com/giongto35/cloud-morph
